docs/modules/databases/influxdb.md

```diff
@@ -1,22 +1,53 @@
 # InfluxDB Module
 
-Testcontainers module for InfluxData [InfluxDB](https://github.com/influxdata/influxdb).
+Testcontainers module for InfluxData [InfluxDB](https://www.influxdata.com/products/influxdb/).
 
-## Usage example
+## Important Note
+
+They are breaking changes in InfluxDB 2.x. For more information refer to the
+main [documentation](https://docs.influxdata.com/influxdb/v2.0/upgrade/v1-to-v2/). InfluxDB
+official [container registry](https://hub.docker.com/_/influxdb) on docker hub.
+
+## InfluxDB 2.x usage example
 
 Running influxDbContainer as a stand-in for InfluxDB in a test:
 
-```java
-public class SomeTest {
-
-    @Rule
-    public InfluxDBContainer influxDbContainer = new InfluxDBContainer();
-    
-    @Test
-    public void someTestMethod() {
-         InfluxDB influxDB = influxDbContainer.getNewInfluxDB();
-         ...
-```
+<!--codeinclude-->
+[InfluxDBContainerTest](../../../modules/influxdb/src/test/java/org/testcontainers/containers/InfluxDBContainerTest.java)
+<!--/codeinclude-->
+
+
+The influxDB will be setup with the following data:<br/>
+
+| Property      | Default Value | 
+| ------------- |:-------------:|
+| username      | test-user     | 
+| password      | test-password | 
+| organization  | test-org      |
+| bucket        | test-bucket   |  
+| retention     | 0 (infinite)  |
+| adminToken    |       -       |
```

Comment:
This only applies to v2, right? we should consider to give visibility about what is set up for each version